Bottom: 33711f70d0a0cdeb023ddcc62e368357705a4a11
Top:    660793628ca46db91ce4f7ac1ec08c6425692ed5
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-05 19:22:08 +0100

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/gtk/object3d.c b/src/hid/gtk/object3d.c
index b355d0e..49462ca 100644
--- a/src/hid/gtk/object3d.c
+++ b/src/hid/gtk/object3d.c
@@ -883,18 +883,6 @@ object3d_from_tracking (void)
     ct = ct->next;
   }
 
-  /* We know how many edges and vertices we need now...
-   *
-   * let n = npoints
-   * bodies = 1             (FOR NOW - just the first board outline)
-   * vertices = 2n          (n-top, n-bottom)
-   * edges = 3n             (n-top, n-bottom, n-sides)
-   * faces = 2 + n          (1-top, 1-bottom, n-sides)
-   *
-   * holes = 0              (FOR NOW - just the outline, no holes)
-   * holes = ncontours - 1  (LATER)
-   */
-
   board_object = make_object3d (PCB->Name);
   board_appearance = make_appearance ();
   top_bot_appearance = make_appearance ();
